src/components/Authenticator.tsx

```diff
@@ -0,0 +1,52 @@
+import { Authenticator as AmplifyAuthenticator } from "@aws-amplify/ui-react";
+import ExtLink from "./ExtLink";
+
+export default function Authenticator() {
+  return (
+    <AmplifyAuthenticator
+      components={{
+        SignUp: {
+          FormFields() {
+            return (
+              <>
+                <AmplifyAuthenticator.SignUp.FormFields />
+                <p className="text-sm">
+                  By signing up, you agree to our{" "}
+                  <ExtLink
+                    href="https://angelgiving.io/privacy-policy/"
+                    className="text-blue hover:text-blue-l2"
```

Comment:
instead of creating new component just to abstract classes, tailwind sometimes recommend use [multi-cursor editing](https://tailwindcss.com/docs/reusing-styles#multi-cursor-editing)